client-feature-matrix/index.mdx

```diff
@@ -14,6 +14,8 @@ import {
   tableview,
 } from "../data/matrix.js";
 
+To get more information on clients, see [Pulsar Client Libraries](/docs/next/client-libraries).
```

Comment:
```suggestion
To get more information on clients, see [Pulsar Client Libraries](/docs/client-libraries).
```

Let's use latest stable version.

Reply:
I tried that, but it results in a 404 error

Reply:
Looks like you can remove the `next` part from the links on the `md` pages but not on the `mdx` pages

Reply:
> it results in a 404 error

The redirect happens in Apache Web Server. So perhaps you need run docker-compose for previewing this manner ..

https://github.com/apache/pulsar-site/blob/80e4090e04dbbbf22652be38d542e4989d8dc3bd/static/.htaccess#L19

https://github.com/apache/pulsar-site/blob/main/docker-compose.yaml

Reply:
Same result using docker compose

Reply:
Well. I'll merge as is now.